### PR TITLE
Ss/feature messages

### PR DESCRIPTION
**Pull request by skopa**
https://trello.com/c/4A950oDm/103-messaging-with-user-layout

### CODE REVIEW

File: app/Http/Controllers/MessageController.php [hunk 1]
```diff
@@ -22,16 +24,43 @@ public function index($userId)
...
+         * $user->whereIn('id', $usersFromIds);
+         * });
+         * })
+         * ->where('id', '!=', 7)
+         * ->get();
+         **/
+
+        $messages = Message::where('user_to_id', $userFrom->id)
+            ->groupBy('user_from_id')
+            ->get();
```
Comment (binaryvo) [inline app/Http/Controllers/MessageController.php]: Please move retrieving messages to an appropriate methoed in Message model

File: resources/assets/js/app/config/debug/config.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 module.exports = {
     debug: true,
-    baseUrl: '/api/v1'
+    baseUrl: '/api/v1',
+    sokcetUrl: 'http://localhost:3000'
```
Comment (binaryvo) [inline resources/assets/js/app/config/debug/config.js]: Looks like a typo

File: resources/assets/js/app/config/prod/config.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 module.exports = {
     debug: false,
-    baseUrl: '/api/v1'
+    baseUrl: '/api/v1',
+    sokcetUrl: 'http://localhost:3000'
```
Comment (binaryvo) [inline resources/assets/js/app/config/prod/config.js]: Looks like a typo: sokcetUrl should be socketUrl

**Comment by skopa** [comment]
corrected

File: resources/assets/js/app/views/messages/messageDialogEditItem.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+    serializeData: function () {
+        var edit = '';
+        if(this.model.get('created_at') != this.model.get('updated_at')) {
+            edit = 'Edit at';
+        }
+
+        return {
+            model: this.model.toJSON(),
+            edit_et: edit
```
Comment (binaryvo) [inline resources/assets/js/app/views/messages/messageDialogEditItem.js]: Typo?

File: resources/assets/js/app/views/messages/messageItem.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+var Marionette = require('backbone.marionette');
+var messageModel = require('../../models/MessageModel');
+
+module.exports = Marionette.ItemView.extend({
+    template: 'messageItem',
+    serializeData: function () {
+        return {
+            model: this.model.toJSON(),
```
Comment (binaryvo) [inline resources/assets/js/app/views/messages/messageItem.js]: Is it model of message? maybe better to rename it to 'message'?

File: resources/assets/js/app/views/messages/messageCollection.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
+var Marionette = require('backbone.marionette');
+var messageItem = require('./messageItem');
+
+module.exports = Marionette.CollectionView.extend({
+    childView: messageItem,
+    childViewOptions : function (model) {
+        var user = this.collection.getMeta().users_from.filter(function (user) {
```
Comment (binaryvo) [inline resources/assets/js/app/views/messages/messageCollection.js]: Would be better to use underscore function findWhere() instead

**Comment by binaryvo** [comment]
just resolve remarks and it will be ready for merge

**Pull request merged**